pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -283,6 +304,13 @@ func (r *ReconcileNodeMaintenance) initMaintenanceStatus(nm *nodemaintenanceapi.
 	return nil
 }
 
+func (r *ReconcileNodeMaintenance) reconcileAndErrorAfter(nm *nodemaintenanceapi.NodeMaintenance, err error) (reconcile.Result, error) {
```

Comment:
Why does this function exist?  It does not appear to be used anywhere?

Reply:
removed